client/i18n/README.md

```diff
@@ -0,0 +1,117 @@
+This project uses `i18next` with `react-i18next` to provide internationalization of content. Human readable content should not be used directly in React components. Instead, they should be looked up using `react-i18next` - which will ensure the content is localized. The library also supports interpolation of dynamic values and formatting of the output. `eslint-plugin-i18next` is used to ensure that JSX cannot be written that contains text without a translation.
+
+## Usage
+
+The `useTranslation` hook provides a `t(key, options)` function - that takes a translation key and (optional) options and returns the translated content. See [https://www.i18next.com/translation-function/essentials](https://www.i18next.com/translation-function/essentials) for more details. An object hash containing values for interpolation can be supplied as the second parameter to `t` instaed of a full options object.
+
+```
+import {useTranslation} from 'react-i18next';
+...
+export const myComponent = () => {
+    const {t} = useTranslation();
+    return (
+        <>
```

Comment:
Empty brackets?

Reply:
I see this is just used to show that there's something wrapping the content, I'm not too fussed about this

Reply:
It's shorthand for `<React.Fragment>` too

Reply:
TIL